explorer/client/src/components/JobRuns/Details.tsx

```diff
@@ -86,15 +86,24 @@ const Details = ({ classes, jobRun }: IProps) => {
         </TableRow>
         <TableRow>
           <KeyCol>Requester</KeyCol>
-          <Col>{jobRun.requester}</Col>
+          <Col>
+            {jobRun.requester && '0x'}
```

Comment:
How do you feel about extracting a presenter function for these? e.g. `withHexPrefix(...)`?